mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,22 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Gitlab Connector UI
+Bundle-SymbolicName: org.eclipse.mylyn.gitlab.ui;singleton:=true
+Bundle-Version: 4.0.0.qualifier
+Export-Package: org.eclipse.mylyn.gitlab.ui
+Bundle-Activator: org.eclipse.mylyn.gitlab.ui.GitlabUiActivator
+Require-Bundle: org.eclipse.ui,
+ org.eclipse.core.runtime,
+ org.eclipse.mylyn.tasks.ui,
+ org.eclipse.mylyn.tasks.core,
+ org.eclipse.ui.forms;bundle-version="3.11.500",
+ org.eclipse.mylyn.gitlab.core;bundle-version="1.0.0",
+ com.google.guava,
```

Comment:
Please prefer `Import-Package` for third party libs

Reply:
Done with next commit